### PR TITLE
1385000 create ef6 data model

### PR DESCRIPTION
**Pull request by john-par**
Update for task to use .NET tutorial template.

### CODE REVIEW

**Comment by john-par** [comment]
@tdykstra this PR is ready for your review. Please let me know what you'd like changed.

**Comment by john-par** [comment]
@tdykstra I forgot to mention that I originally removed the *add-scaffolding.png* from the article and the repo. The first time I submitted the PR, I got a warning because another article in a different area of the repo was referencing the PNG. I replaced the PNG, but I did not add it back to this article.

**Comment by tdykstra** [comment]
@john-par Sounds good, thanks.

**Comment by john-par** [comment]
@tdykstra I'm sorry for the confusion. I have implemented the feedback from your comments and I reverted all changes that were not required by the User Story. Please let me know if you'd like any other changes made.

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application.md [hunk 1]
```diff
@@ -310,15 +280,32 @@ The amount of code you had to write in order for Entity Framework to be able to
...
 
-You've seen that conventions can be overridden. For example, you specified that table names shouldn't be pluralized, and you'll see later how to explicitly mark a property as a foreign key property. You'll learn more about conventions and how to override them in the [Creating a More Complex Data Model](creating-a-more-complex-data-model-for-an-asp-net-mvc-application.md) tutorial later in this series. For more information about conventions, see [Code First Conventions](/ef/ef6/modeling/code-first/conventions/built-in).
+You've seen that conventions can be overridden. For example, you specified that table names shouldn't be pluralized, and you'll see later how to explicitly mark a property as a foreign key property.
+## Additional resources
+
+For more about EF 6, see these articles:
```
Comment (tdykstra) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application.md]: After "see these articles" you'd expect to find a list of articles, but the bullets start "you can find...", "To learn more ..." etc.  I think the list can stand alone as is without this intro line.

**Comment by john-par** [comment]
@tdykstra your feedback has been incorporated. This PR is ready to be merged.

**Pull request merged**